airbyte-integrations/connectors/source-it-glue/sample_files/configured_catalog.json

```diff
@@ -0,0 +1,464 @@
+{
+    "streams": [
+        {
+            "stream": {
+                "name": "contacts",
+                "json_schema": {
+                    "$schema": "http://json-schema.org/draft-04/schema#",
+                    "type": "object",
+                    "properties": {
+                        "token": {
+                            "type": "string"
+                        }
+                    }
```

Comment:
I don't think we are using this (isn't it same as the other configured_catalog file), If so can we delete this?

Reply:
@harshithmullapudi I will delete the sample_files folder :)